modules/hal_nordic/nrfx/Kconfig

```diff
@@ -25,6 +25,10 @@ config NRFX_COMP
 	bool "COMP driver"
 	depends on $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF_COMP))
 
+config NRFX_CRACEN
+	bool "CRACEN drivers"
+	depends on SOC_COMPATIBLE_NRF54LX
```

Comment:
Should depend on DT.

Reply:
There is no device tree node today for cracen itself. I could create one, but it really is just extra steps.